<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>searchForm</class>
 <widget class="QMainWindow" name="searchForm">
  <property name="windowModality">
   <enum>Qt::WindowModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1200</width>
    <height>500</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>1200</width>
    <height>0</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>SEARCH</string>
  </property>
  <property name="windowIcon">
   <iconset resource="custom.qrc">
    <normaloff>:/icons/images/icons/search.png</normaloff>:/icons/images/icons/search.png</iconset>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <property name="topMargin">
       <number>0</number>
      </property>
      <item>
       <widget class="QLineEdit" name="lineEditItemSearch"/>
      </item>
      <item>
       <widget class="QComboBox" name="comboBoxItemSearch">
        <item>
         <property name="text">
          <string>ID</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>MAKE &amp; MODEL</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>IMEI/S.No</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>KNOWN FAULTS</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>ADDITIONAL DETAILS</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>BILL</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>AMOUNT PAID</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>STATUS</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>DATE IN</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>DATE OUT</string>
         </property>
        </item>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QTableWidget" name="tableWidgetItemSearch">
      <property name="selectionMode">
       <enum>QAbstractItemView::SingleSelection</enum>
      </property>
      <property name="selectionBehavior">
       <enum>QAbstractItemView::SelectRows</enum>
      </property>
      <column>
       <property name="text">
        <string>ID</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>MAKE &amp; MODEL</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>IMEI/S. No</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>KNOWN FAULTS</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>ADDITIONAL DETAILS</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>BILL</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>AMOUNT PAID</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>STATUS</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>DATE IN</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>DATE OUT</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>CUSTID</string>
       </property>
      </column>
     </widget>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <property name="topMargin">
       <number>0</number>
      </property>
      <item>
       <widget class="QLineEdit" name="lineEditCustomerSearch"/>
      </item>
      <item>
       <widget class="QComboBox" name="comboBoxCustomerSearch">
        <item>
         <property name="text">
          <string>ID</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>NAME</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>MOBILE</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>DATE ADDED</string>
         </property>
        </item>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QTableWidget" name="tableWidgetCustomerSearch">
      <property name="selectionMode">
       <enum>QAbstractItemView::SingleSelection</enum>
      </property>
      <property name="selectionBehavior">
       <enum>QAbstractItemView::SelectRows</enum>
      </property>
      <column>
       <property name="text">
        <string>ID</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>NAME</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>MOBILE</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>MOBILE 2</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>DATE ADDED</string>
       </property>
      </column>
     </widget>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_3">
      <property name="topMargin">
       <number>0</number>
      </property>
      <item>
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QPushButton" name="pushButtonAddItem">
        <property name="minimumSize">
         <size>
          <width>30</width>
          <height>30</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>30</width>
          <height>30</height>
         </size>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="icon">
         <iconset resource="custom.qrc">
          <normaloff>:/icons/images/icons/plus.svg</normaloff>:/icons/images/icons/plus.svg</iconset>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pushButtonSave">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>30</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>30</width>
          <height>30</height>
         </size>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="icon">
         <iconset resource="custom.qrc">
          <normaloff>:/icons/images/icons/save.png</normaloff>:/icons/images/icons/save.png</iconset>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1200</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources>
  <include location="custom.qrc"/>
 </resources>
 <connections/>
</ui>
